src/rapids_dependency_file_generator/rapids_dependency_file_generator.py

```diff
@@ -3,33 +3,26 @@
 import os
 import textwrap
 from collections import defaultdict
+from collections.abc import Generator
```

Comment:
This should probably be `typing.Generator` rather than an ABC. It's just used for a type hint.

https://docs.python.org/3/library/typing.html#typing.Generator

```suggestion
from typing import Generator
```

Reply:
From https://docs.python.org/3/library/typing.html#typing.Generator:

> Deprecated alias to [`collections.abc.Generator`](https://docs.python.org/3/library/collections.abc.html#collections.abc.Generator).

Reply:
Wow, if only I could read the link that I shared. :) Resolving this.